### PR TITLE
docs(intro+update): add WLS 14 support

### PR DESCRIPTION
**Pull request by tmetzke**
related to CAM-13590

### CODE REVIEW

**Review by tasso94 — APPROVED** [review]
LGTM!

File: content/update/minor/716-to-717/wls.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ After shutting down the server, replace the following libraries in `$WLS_DOMAIN_
 * `camunda-commons-utils-$COMMONS_VERSION.jar`
 * `camunda-connect-connectors-all-$CONNECT_VERSION.jar`
 * `camunda-connect-core-$CONNECT_VERSION.jar`
-* `camunda-template-engines-freemarker-$TEMPLATE_ENGINES_VERSION.jar`
 * `feel-engine-$FEEL_ENGINE_VERSION-scala-shaded.jar`
-* `freemarker-$FREEMARKER_VERSION.jar`
```
[the remark this answers is not in the document]
Comment (tmetzke) [reply content/update/minor/716-to-717/wls.md:54]: Looking at the `7.16` and `7.17` artifacts for WLS, those are not present. I didn't go further back, but I could imagine we never had those in our WLS libs.

[the remark this answers is not in the document]
Comment (tasso94) [reply content/update/minor/716-to-717/wls.md:54]: That's alright. We shouldn't list them in this case. 👍

[the remark this answers is not in the document]
Comment (tmetzke) [reply content/update/minor/716-to-717/wls.md:54]: 👍

**Pull request merged**